Task_10/style1.css

```diff
@@ -0,0 +1,551 @@
+       
+    }}
+
+@media ( max-width:800px){
+   /* гамбургер меню/////// */
+   .hamburger-menu{
+    display: flex;
+}
+#menu__toggle {
+    opacity: 0;
+  }
+  
+  #menu__toggle:checked ~ .menu__btn > span {
+    transform: rotate(45deg);
+  }
+  #menu__toggle:checked ~ .menu__btn > span::before {
+    top: 0;
+    transform: rotate(0);
+  }
+  #menu__toggle:checked ~ .menu__btn > span::after {
+    top: 0;
+    transform: rotate(90deg);
+  }
+  #menu__toggle:checked ~ .menu__box {
+    visibility: visible;
+    left: 0;
+  }
+  
+  .menu__btn {
+    display: flex;
+    align-items: center;
+    position: fixed;
+    top: 20px;
+    left: 80%;
+  
+    width: 26px;
+    height: 26px;
+   
+  
+    cursor: pointer;
+    z-index: 1;
+  }
+  
+  .menu__btn > span,
+  .menu__btn > span::before,
+  .menu__btn > span::after {
+    display: block;
+    position: absolute;
+    right: 0;
+  
+    width: 100%;
+    height: 2px;
+  
+    background-color: #3b3834fd;
+  
+    transition-duration: .25s;
+  }
+  .menu__btn > span::before {
+    content: '';
+    top: -8px;
+  }
+  .menu__btn > span::after {
+    content: '';
+    top: 8px;
+  }
+  
+  .menu__box {
+    display: block;
+    position: fixed;
+    visibility: hidden;
+    top: 0;
+    left: -100%;
+  
+    width: 300px;
+    height: 100%;
+  
+    margin: 0;
+    padding: 80px 0;
+  
+    list-style: none;
+  
+    background-color:  #f39509e3;
+    box-shadow: 1px 0px 6px rgba(0, 0, 0, .2);
+  
+    transition-duration: .25s;
+  }
+  
+  .menu__item {
+    display: block;
+    padding: 12px 24px;
+  
+    color: #333;
+  
+    font-family: BebasNeueRegular; 
+    font-size: 30px;
+    font-weight: 400;
+  
+    text-decoration: none;
+  
+    transition-duration: .25s;
+  }
+  .menu__item:hover {
+    background-color: #CFD8DC;
+  }
+
+/* ///////////////////////// */}
+    
+.orange{
+    color: #FBA929;
+    letter-spacing: 1px;
+    padding-left: 8px;
+    
+}
+
+.rabbit_text4{
+    color: rgba(59, 59, 59, 0.938);
+
+}
+/* ////////////////// */
+.container{
+    display:flex;
+    border: solid#FBA929  ;
+    width: auto;
+    height: 400px; 
+    background-image: url(./img/big-orange-bg.png);
+    margin-top: 1%;
+    background-size: cover;
+    background-repeat: no-repeat;
+    
+
+}
+.flex_text1{
+    display: flex;
+    flex-direction: column;
+    align-items:center;
+    justify-content: flex-start;
+    margin-left: 280px;
+    padding-top: 100px;
+    font-size: 130%;
+    
+
+}
+@media ( max-width:800px){
+    .flex_text1{
+        display: flex;
+        margin-left: 50%;
+        padding-top: 0;
+        justify-content: center;
+}}
+.left1{
+    position: absolute; 
+    color:orange;
+    width: 400px;
+    background-color:rgba(29, 28, 28, 0.884);
+}
+
+.left2{
+    position: absolute; 
+    color:black;
+    background-color:orange;
+    width: 400px;
+    margin-top: 100px;
+}
+
+@media ( max-width:800px){
+    .left1{
+        width:250px;
+        font-size: 95%;
```

Comment:
Pleeeease don't use `%` without the need. Use `px`

Reply:
fixed